CHANGELOG.md

```diff
@@ -6,10 +6,13 @@
 * Fixed issue with Clustergram rows and columns reordering incorrectly
   on the heatmap when precomputed traces are used to generate the
   figure.
-* Fixed issue with Speck not rendering unless it is attached to a callback.
+* Fixed issue with Speck not rendering unless it is attached to a
+  callback.
 * Prevent Speck from trying to calculate a system with no atom.
 * Let Clustergram render even when row and column labels are not
   specified, by adding default values.
+* Allow multiple SequenceViewer components to render on the same
```

Comment:
```suggestion
* Allowed multiple SequenceViewer components to render on the same
```
Alternatively, change all preterit verbs from previous entries... (which was originally my preference, if you recall)